tools/local-dev.sh

```diff
@@ -17,6 +17,9 @@ bazelCommand=${BAZEL:-"yarn bazel"}
   ${bazelCommand} build :npm_package
 )
 
+export TS_NODE_TRANSPILE_ONLY=1
```

Comment:
We don't have to export this right? Wouldn't have pollute the environment for all subsequent runs?

Reply:
This script is never sourced, so it should only be local to this subshell.

Reply:
Okay, just wanted to be sure.